### ISSUE TITLE
unable to upgrade from .75 to .8

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nicholasmr/obblm#459]
```
What steps will reproduce the problem?
1. running 0.75 to 0.8 upgrade script 

What is the expected output? What do you see instead?

Expected output is a an all clear on OKs for each table conversion/creation. 
Instead I get the following:

"Running SQLs for core system upgrade...
OK — Synchronized game data with database
This function has none of DETERMINISTIC, NO SQL, or READS SQL DATA in its 
declaration and binary logging is enabled (you *might* want to use the less 
safe log_bin_trust_function_creators variable) CODE: ----- CREATE FUNCTION 
syncELOMatch(trid MEDIUMINT UNSIGNED, mid MEDIUMINT SIGNED) RETURNS BOOLEAN NOT 
DETERMINISTIC CONTAINS SQL BEGIN DECLARE Rt1_0, Rt2_0, Rc1_0, Rc2_0, Rt1, Rt2, 
Rc1, Rc2 FLOAT; ...."

Basically, it seems like the upgrade is conking out when it gets to CREATE 
FUNCTION syncELOMatch.

I'm running MySQL 5.0.91-community-log on the production server and 5.0.51a on 
the dev server. Upgrade fails on both.
```

Original issue reported on code.google.com by `kavi...@gmail.com` on 18 Aug 2010 at 5:48

Attachments:
- [OBBLMupgradeOutput.htm](https://storage.googleapis.com/google-code-attachments/obblm/issue-459/comment-0/OBBLMupgradeOutput.htm)
